onnx/test/hub_test.py

```diff
@@ -56,7 +56,7 @@ def test_custom_cache(self):  # type: () -> None
         hub.set_dir(old_cache)
 
     def test_download_with_opset(self):  # type: () -> None
-        model = hub.load(self.name, self.repo, opset=12)
+        model = hub.load(self.name, self.repo, opset=8)
```

Comment:
what does this fix? Is this required because the test model was changed to MNIST?

Reply:
Yes. I had this in my local branch but forgot to add it in last PR. My bad!

Reply:
FYI: https://github.com/onnx/models/tree/master/vision/classification/mnist Update it as 8 since there is no 12 for MNIST.